src/rust/src/packed_r_tree.rs

```diff
@@ -188,7 +188,7 @@ async fn read_http_node_items(
 pub struct SearchResultItem {
     /// Byte offset in feature data section
     pub offset: usize,
-    /// Feature number
+    /// Feature number (REVIEW: why do we need this?)
```

Comment:
Checked it, it's only used in packed_r_tree tests. It stems most probably from a direct translation of the original C++ implementation.